akka-docs/src/test/java/jdocs/stream/IntegrationDocTest.java

```diff
@@ -693,5 +695,28 @@ public void illustrateOrderingAndParallelismOfMapAsyncUnordered() throws Excepti
     };
   }
 
+  @Test
+  public void illustrateSourceQueue() throws Exception {
+    new TestKit(system) {
+      {
+        //#source-queue
+        int bufferSize = 5;
+        int elementsToProcess = 3;
+
+        SourceQueueWithComplete<Integer> sourceQueue = Source.<Integer>queue(bufferSize, OverflowStrategy.backpressure())
+            .throttle(elementsToProcess, Duration.ofSeconds(3), elementsToProcess, ThrottleMode.shaping())
+            .map(x -> x * x)
+            .to(Sink.foreach(x -> System.out.println("completed " + x)))
```

Comment:
perhaps not use the word "completed" since we typically use that when the stream is completed
"processed", or "got" ?

Reply:
thanks!